mlflow/recipes/utils/metrics.py

```diff
@@ -42,20 +42,20 @@ def from_custom_metric_dict(cls, custom_metric_dict):
     RecipeMetric(name="false_positives", greater_is_better=False),
     RecipeMetric(name="false_negatives", greater_is_better=False),
     RecipeMetric(name="true_positives", greater_is_better=True),
-    RecipeMetric(name="recall_score", greater_is_better=True),
-    RecipeMetric(name="precision_score", greater_is_better=True),
+    RecipeMetric(name="recall", greater_is_better=True),
+    RecipeMetric(name="precision", greater_is_better=True),
     RecipeMetric(name="f1_score", greater_is_better=True),
-    RecipeMetric(name="accuracy_score", greater_is_better=True),
+    RecipeMetric(name="accuracy", greater_is_better=True),
     RecipeMetric(name="roc_auc", greater_is_better=True),
+    RecipeMetric(name="log_loss", greater_is_better=False),
 ]
 
 BUILTIN_MULTICLASS_CLASSIFICATION_RECIPE_METRICS = [
-    RecipeMetric(name="recall_score", greater_is_better=True),
-    RecipeMetric(name="precision_score", greater_is_better=True),
+    RecipeMetric(name="recall", greater_is_better=True),
+    RecipeMetric(name="precision", greater_is_better=True),
     RecipeMetric(name="f1_score_macro", greater_is_better=True),
     RecipeMetric(name="f1_score_micro", greater_is_better=True),
-    RecipeMetric(name="accuracy_score", greater_is_better=True),
-    RecipeMetric(name="roc_auc_ovr", greater_is_better=True),
```

Comment:
Could you file a ticket to add `roc_auc_score` to both mlflow.evaluate() and recipes? This is a useful metric for classification. We should support it: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html

Reply:
Seems like we already support it. So added it back :) 
It is just named differently. 
https://github.com/mlflow/mlflow/blob/a2783d058d2f3151421565af2146bd4f455acfcf/mlflow/models/evaluation/default_evaluator.py#L237

Reply:
This is great!